lms/templates/instructor/instructor_dashboard_2/course_info.html

```diff
@@ -44,24 +44,24 @@ <h4 class="hd hd-4">${_("Enrollment Information")}</h4>
   <h4 class="hd hd-4">${_("Basic Course Information")}</h4>
 
   <ul class="list-input">
-    <li class="field text is-not-editable" id="field-course-organization">
-      <label for="course-organization">${_("Organization:")}</label>
-      <b>${ section_data['course_id'].org}</b>
+    <li class="field text is-not-editable" id="field-course-display-name">
+      <label for="course-display-name">${_("Course Name:")}</label>
+      <b>${ section_data['course_display_name']}</b>
     </li>
 
     <li class="field text is-not-editable" id="field-course-number">
       <label for="course-number">${_("Course Number:")}</label>
-      <b>${ section_data['course_id'].course}</b>
+      <b>${ section_data['course_number']}</b>
     </li>
 
     <li class="field text is-not-editable" id="field-course-name">
-      <label for="course-name">${_("Course Name:")}</label>
-      <b>${ section_data['course_id'].run}</b>
+      <label for="course-name">${_("Course Run:")}</label>
+      <b>${ section_data['course_id'].run }</b>
     </li>
 
-    <li class="field text is-not-editable" id="field-course-display-name">
-      <label for="course-display-name">${_("Course Display Name:")}</label>
-      <b>${ section_data['course_display_name']}</b>
+    <li class="field text is-not-editable" id="field-course-organization">
+      <label for="course-organization">${_("Organization:")}</label>
+      <b>${ section_data['course_id'].org }</b>
```

Comment:
I think these fields should be in order of decreasing specificity:
* Course name (display name)
* Course Run
* Course Number
* Organization